pkg/services/apiserver/options/storage.go

```diff
@@ -21,17 +22,20 @@ const (
 type StorageOptions struct {
 	StorageType StorageType
 	DataPath    string
+	Address     string
 }
 
 func NewStorageOptions() *StorageOptions {
 	return &StorageOptions{
 		StorageType: StorageTypeLegacy,
+		Address:     "localhost:10000",
 	}
 }
 
 func (o *StorageOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar((*string)(&o.StorageType), "grafana-apiserver-storage-type", string(o.StorageType), "Storage type")
 	fs.StringVar((*string)(&o.StorageType), "grafana-apiserver-storage-path", string(o.StorageType), "Storage path for file storage")
+	fs.StringVar((*string)(&o.Address), "grafana-apiserver-storage-address", string(o.Address), "Remote grpc address endpoint")
```

Comment:
@chaudyg what are your thoughts on making independent options for unified storage and composing them in storage options? something similar to [recommended options](https://github.com/kubernetes/apiserver/blob/133975e0c8bd99f24f9f690c9616cfdb64630a16/pkg/server/options/recommended.go#L88) in the apiserver library

Reply:
Could you give  me more details ?